go/vt/vtgate/planbuilder/testdata/tpch_cases.json

```diff
@@ -746,12 +746,12 @@
   {
     "comment": "TPC-H query 8",
     "query": "select o_year, sum(case when nation = 'BRAZIL' then volume else 0 end) / sum(volume) as mkt_share from ( select extract(year from o_orderdate) as o_year, l_extendedprice * (1 - l_discount) as volume, n2.n_name as nation from part, supplier, lineitem, orders, customer, nation n1, nation n2, region where p_partkey = l_partkey and s_suppkey = l_suppkey and l_orderkey = o_orderkey and o_custkey = c_custkey and c_nationkey = n1.n_nationkey and n1.n_regionkey = r_regionkey and r_name = 'AMERICA' and s_nationkey = n2.n_nationkey and o_orderdate between date '1995-01-01' and date('1996-12-31') and p_type = 'ECONOMY ANODIZED STEEL' ) as all_nations group by o_year order by o_year",
-    "plan": "VT12001: unsupported: in scatter query: complex aggregate expression"
+    "plan": "VT12001: unsupported: complex aggregation expression: sum(case when nation = 'BRAZIL' then volume else 0 end)"
   },
   {
     "comment": "TPC-H query 9",
     "query": "select nation, o_year, sum(amount) as sum_profit from ( select n_name as nation, extract(year from o_orderdate) as o_year, l_extendedprice * (1 - l_discount) - ps_supplycost * l_quantity as amount from part, supplier, lineitem, partsupp, orders, nation where s_suppkey = l_suppkey and ps_suppkey = l_suppkey and ps_partkey = l_partkey and p_partkey = l_partkey and o_orderkey = l_orderkey and s_nationkey = n_nationkey and p_name like '%green%' ) as profit group by nation, o_year order by nation, o_year desc",
-    "plan": "VT12001: unsupported: aggregation on columns from different sources"
+    "plan": "VT12001: unsupported: complex aggregation expression: sum(amount) as sum_profit"
```

Comment:
this is an example of what I wrote above - `sum(amount)` is not a complex aggregation, IMO, so this error message is confusing at best